apps/api/src/app/workflows-v2/mappers/notification-template-mapper.ts

```diff
@@ -16,29 +18,30 @@ import { GetPreferencesResponseDto } from '@novu/application-generic';
 import { buildSlug } from '../../shared/helpers/build-slug';
 
 export function toResponseWorkflowDto(
-  template: NotificationTemplateEntity,
+  persistedWorkflow: NotificationTemplateEntity,
   preferences: GetPreferencesResponseDto | undefined
 ): WorkflowResponseDto {
   const preferencesDto: PreferencesResponseDto = {
     user: preferences?.source[PreferencesTypeEnum.USER_WORKFLOW] || null,
     default: preferences?.source[PreferencesTypeEnum.WORKFLOW_RESOURCE] || DEFAULT_WORKFLOW_PREFERENCES,
   };
-  const workflowName = template.name || 'Missing Name';
+  const workflowName = persistedWorkflow.name || 'Missing Name';
 
   return {
-    _id: template._id,
-    slug: buildSlug(workflowName, ShortIsPrefixEnum.WORKFLOW, template._id),
-    workflowId: template.triggers[0].identifier,
+    _id: persistedWorkflow._id,
+    slug: buildSlug(workflowName, ShortIsPrefixEnum.WORKFLOW, persistedWorkflow._id),
+    workflowId: persistedWorkflow.triggers[0].identifier,
     name: workflowName,
-    tags: template.tags,
-    active: template.active,
+    tags: persistedWorkflow.tags,
+    active: persistedWorkflow.active,
     preferences: preferencesDto,
-    steps: getSteps(template),
-    description: template.description,
-    origin: computeOrigin(template),
-    updatedAt: template.updatedAt || 'Missing Updated At',
-    createdAt: template.createdAt || 'Missing Create At',
+    steps: getSteps(persistedWorkflow),
+    description: persistedWorkflow.description,
+    origin: computeOrigin(persistedWorkflow),
+    updatedAt: persistedWorkflow.updatedAt || 'Missing Updated At',
```

Comment:
These fallback values on timestamps can be removed as createdAt and UpdatedAt are always added.

Reply:
and yet they are undefined on the entity level, better have that than a question mark, it's not new code, reverting the change here